src/Components/ReferenceTokenStatus.razor

```diff
@@ -0,0 +1,21 @@
+@using IdentityServerConfig.Models
+
+@if (!ReferenceTokenData.Status.Equals(ReferenceTokenDataModel.StatusCode.NotChecked))
+{
+    <div class="alert @(ReferenceTokenData.Status.Equals(ReferenceTokenDataModel.StatusCode.Active) ? "alert-success" : "alert-danger") " role="alert">
+        <h4 class="alert-heading">@ReferenceTokenData.Status.ToString()</h4>
+        @if (ReferenceTokenData.Status.Equals(ReferenceTokenDataModel.StatusCode.Invalid))
+        {
+            <p>The Reference Token for this client is invalid</p>
+        }
+        else
+        {
+            <p>Creation Time: @ReferenceTokenData.CreationTime UTC</p>
+            <p>Expiration Time: @ReferenceTokenData.Expiration UTC</p>
+        }
+    </div>
+}
+@code{
+    [Parameter]
+    public ReferenceTokenDataModel ReferenceTokenData { get; set; }
```

Comment:
Maybe we should rename this property to just `Model`. Makes this code a bit cleaner and easier to read IMHO.

Reply:
Resolved in 6b019ab